velox/type/Variant.cpp

```diff
@@ -221,11 +221,18 @@ std::string variant::toJson(const TypePtr& type) const {
       std::string b{};
       b += "[";
       bool first = true;
+      uint32_t idx = 0;
```

Comment:
We should check the number of children match as well along with a test.

Reply:
Added the checks in toJson() function.